### ISSUE TITLE
Update .deb dependencies

### ISSUE THREAD

**Issue by ConorIA** [wireapp/wire-desktop#171]
I don't know for sure, but I imagine that the .deb has depencies outside of the default libappindicator stuff. On Arch, these are the depencies, as detected by pamac.

```
alsa-lib (alsa-lib-noassertion)
gconf (gconf-gtk2)
gtk2 (gtk2-aqd, gtk2-patched-filechooser-icon-view, gtk2-patched-gdkwin-nullcheck, gtk2-ubuntu)
libxss
libxtst
nss
```

Does Debian have a tool to detect dependencies? Or can a Debian user try running Wire in a clean chroot? 

I guess this isn't a huge issue, since no one has complained so far, but it might be worth it to have the metadata complete.

**Comment by sonicou** [comment]
when installing wire on a minimal debian jessie system the following packages need to be installed:

libappindicator1
libnotify-bin

both of them will install a large list of dependencies themselfs (like gtk)

**Comment by ConorIA** [comment]
I've installed the deb in a Docker container using the official ubuntu:latest Docker image. The following are missing dependencies: 

| Missing Shared Library | Package to Install |
|--------------------------------|--------------------------|
| libXss.so.1 | libxss1 |
| libgconf-2.so.4 | libgconf-2-4 |
| libnss3.so | libnss3 |
| libasound.so.2 | libasound2 |

**Comment by sonicou** [comment]
I'm not so sure their docker image is a minimal ubuntu system or is it?

**Comment by ConorIA** [comment]
Yes, very minimal. In fact, the missing deps might even be part of the default desktop install, but I figure it is best to be thorough.

**Comment by lipis** [comment]
Solved in #188
